config/ProjectConfiguration.class.php

```diff
@@ -24,6 +24,8 @@
 
 require_once __DIR__.'/../lib/QubitApcUniversalClassLoader.php';
 
+require_once __DIR__.'/../vendor/composer/autoload.php';
+
```

Comment:
Now that this is added to the main project configuration, I think it could be removed from other places (like lib/CsvImportValidator.class.php).